src/api/app/views/webui/project/_basic_info.html.haml

```diff
@@ -11,3 +11,9 @@
       %i No description set
     - else
       = render partial: 'webui/shared/collapsible_text', locals: { text: project.description }
+
+  - if policy(project).update?
+    .pt-4
+      = link_to(edit_project_path(project), class: 'nav-link', remote: true, title: 'Edit Project') do
```

Comment:
Sorry, I didn't see that in my last review...

```suggestion
      = link_to(edit_project_path(project), class: 'nav-link', remote: true, title: 'Edit') do
```

Reply:
My fault